timescaledb/tutorials/grafana/grafana-timescalecloud.md

```diff
@@ -7,18 +7,21 @@ keywords: [Grafana, visualizations, analytics]
 import CloudTrial from 'versionContent/_partials/_cloudtrial.mdx';
 
 # Connect TimescaleDB and Grafana
+
 Grafana ships with built-in Prometheus, PostgreSQL, Jaeger, and other data
 source plugins that allow you to query and visualize data from a compatible
 database. To add a data source in Grafana you must be signed as a user with
 organization administration role privileges.
 
-To connect Grafana with TimescaleCloud database instance, install Grafana. For
-information about installing Grafana, see the [Grafana installation documentation][grafana-install].
+To connect Grafana with TimescaleCloud database instance, create a
+[Grafana][install-grafana] service on [MST][mst-login]. You can try it for
```

Comment:
Wait - Cloud or MST?

Reply:
This is about the [Grafana service on MST](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/installation/#create-a-new-service-for-grafana)

Reply:
In that case, you need to say MST on L16.

Reply:
> In that case, you need to say MST on L16.

Yes!!! makes sense. Thanks for pointing it.